ecllibrary/std/File.ecl

```diff
@@ -414,27 +414,28 @@ EXPORT DfuPlusExec(varstring cmdline) :=
  * @param compress      Whether to compress the new file. Defaults to FALSE.
  * @param failIfNoSourceFile If TRUE it causes a missing source file to trigger a failure.  Defaults to FALSE.
  * @param expireDays    Number of days to auto-remove file. Default is -1, not expire.
+ * @param dfuServerQueue Name of target DFU Server queue.
```

Comment:
Worth saying what the default is here (blank?) and what it means (1st dfuserver in environment?)